go/host/host.go

```diff
@@ -146,7 +146,7 @@ func (a *Node) Start() {
 			AggregatorID:  &a.ID,
 			Attestation:   encodedAttestation,
 			InitialSecret: a.EnclaveClient.GenerateSecret(),
-			HostAddress:   a.config.P2PAddress,
+			HostAddress:   a.config.P2PBindAddress,
```

Comment:
shouldn't the p2p public address be advertised here?

Reply:
Yep, fixed :)